.eslintrc

```diff
@@ -48,6 +48,10 @@
     "no-throw-literal": 2,
     "no-trailing-spaces": 2,
     "no-unused-expressions": 0,
+    "no-unused-vars": [2, {
+      "argsIgnorePattern": "^var_|opt_|unused_|_",
```

Comment:
unusedArg is fine with me.
On Dec 23, 2015 9:47 AM, "erwin mombay" <notifications@github.com> wrote:

> In .eslintrc
> <https://github.com/ampproject/amphtml/pull/1220#discussion_r48365467>:
>
> > @@ -48,6 +48,10 @@
> >      "no-throw-literal": 2,
> >      "no-trailing-spaces": 2,
> >      "no-unused-expressions": 0,
> > +    "no-unused-vars": [2, {
> > +      "argsIgnorePattern": "^var_|opt_|unused_|_",
>
> so, the google-camelcase plugin unfortunately starts erroring out if we
> used unused_, are we ok with doing unusedArg or var_unused_
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ampproject/amphtml/pull/1220/files#r48365467>.
>